src/core/public/chrome/nav_links/nav_links_service.test.ts

```diff
@@ -41,7 +42,7 @@ const availableApps = new Map([
       id: 'app2',
       order: -10,
       title: 'App 2',
-      euiIconType: 'canvasApp',
+      euiIconType: 'canvasApp' as EuiIconType,
```

Comment:
Yeah, I forgot that those are really assertions, not casts 😢 . The only one that makes sense to assert is the one test where we are intentionally using a made-up string rather than a valid icon name. Fixed.